[Modified] React Wizard (#6355)
getsentry/sentry-docs: src/wizard/javascript/react.md

npm install --save @sentry/react @sentry/tracing
```

## Configure

> Initialise Sentry as early as possible in your application's lifecycle.

Reviewer: Can we combine this sentence with the next one, it seems ambiguous.

Author: @HazAT let me know if I've misunderstood what you meant

From the usability tests I've witnessed, engineers tend to copy these chunks of code without thinking. So whilst I would understand the drive to simply 3 steps into 2, I assume it would likely lead to more human error.

Next, import and initialize the Sentry module as early as possible, before initializing React:

```javascript
import { createRoot } React from "react-dom/client";
import React from "react";   
import * as Sentry from "@sentry/react"; 
import { BrowserTracing } from "@sentry/tracing"; 
import App from "./App"; 

Sentry.init({ 
  dsn: "___PUBLIC_DSN___", 
  integrations: [new BrowserTracing()], 

>   tracesSampleRate: 1.0, 

Reviewer: If we wait for the next JS release which should be next week, we could use 
```suggestion
  enableTracing: true, 
```

}); 

const container = document.getElementById(“app”);
const root = createRoot(container);
root.render(<App />)
```

<div class="alert alert-info" role="alert">
	<h5 class="no_toc"><code>tracesSampleRate: 1.0</code></h5>
	<div class="alert-body content-flush-bottom">

> 		The example above ensures every transaction will be to Sentry, but we recommend lowering this value in production.  

Author: The only sentence that I contrived to help emphasize the code example

	</div>
</div>

## Verify

> This snippet includes an intentional error, so you can test that everything is working as soon as you set it up.

Reviewer 2: ```suggestion
This snippet contains an intentional error and can be used as a test to make sure that everything's working as it should be.
```

```javascript
return <button onClick={() => methodDoesNotExist()}>Break the world</button>;
```
---

> ## Next Steps 

Reviewer 2: ```suggestion
## Additional Info 
```
